viewer/packages/camera-manager/src/Flexible/FlexibleControlsTranslator.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { OrthographicCamera, PerspectiveCamera, Plane, Raycaster, Vector3 } from 'three';
+import { FlexibleControls } from './FlexibleControls';
+
+/**
+ * @beta
+ * This class handles the translation of the camera.
+ * It tries to translate the camera so the same (x,y, z) point is below the mouse
+ */
+export class FlexibleControlsTranslator {
+  //================================================
+  // INSTANCE FIELDS
+  //================================================
+
+  private readonly _controls: FlexibleControls;
+  private readonly _prevPickedPoint = new Vector3();
+  private readonly _plane = new Plane();
+  private readonly _raycaster = new Raycaster();
+  private _camera: PerspectiveCamera | OrthographicCamera | undefined;
```

Comment:
We do not support `OrthographicCamera` camera type

Reply:
For the future. It was like that before I started